.gitignore

```diff
@@ -0,0 +1,18 @@
+.env
+.vscode
+.expo
+.DS_Store
+
+backend/__pycache__
+backend/routers/__pycache__
+backend/models/__pycache__
+backend/services/__pycache__
+backend/.venv
+backend/logs/*
+!backend/logs/.gitkeep
+backend/logs/*
+!backend/logs/.gitkeep
+
+frontend/.env
```

Comment:
Can we add an ignore for any angular builds. We might want to build the frontend separate at some point and it wouldn't need to be pushed to the remote.

Reply:
Resolved. Updated the gitignore.